src/community/oseo/oseo-service/src/test/java/org/geoserver/opensearch/eo/SearchTest.java

```diff
@@ -177,6 +177,7 @@ public void testAllCollection() throws Exception {
     @Test
     public void testAllCollectionCountZero() throws Exception {
         MockHttpServletResponse response = getAsServletResponse("oseo/search?count=0&httpAccept=" + ENCODED_ATOM_MIME);
+        System.out.println(response.getContentAsString());
```

Comment:
System.out here. Is it ok? or is it a leftover from some debug?

Reply:
Whoops yes, removing